api/tests/routes/adage_iframe/get_collective_offer_template_test.py

```diff
@@ -83,8 +86,32 @@ def test_get_collective_offer_template(self, client):
             "imageUrl": None,
             "imageCredit": None,
             "nationalProgram": {"id": national_program.id, "name": national_program.name},
+            "isFavorite": True,
         }
 
+    def test_is_not_a_redactors_favorite(self, client):
+        """Ensure that the isFavorite field is true only if the
+        redactor added it to its favorites.
+        """
+        # not a redactor's favorite offer
```

Comment:
Cette ligne répète un peu ce qui est déjà dans le commentaire au dessus, et dans le nom du test; `# not a redactor's favorite offer` n'est pas nécessaire à mon humble avis